### PR TITLE
SendingLightning: Display payment paths after successful lightning payment

### PR DESCRIPTION
**Pull request by shubhamkmr04**
https://github.com/ZeusLN/zeus/issues/1927

Simulator Screenshot - iPhone 15 Pro - 2024-08-06 at 22 58 12
Simulator Screenshot - iPhone 15 Pro - 2024-08-06 at 22 58 22

### CODE REVIEW

**Comment by kaloudis** [comment]
Maybe we should join the payment path and add note buttons on one line to save real estate.

[Payment Path (3)] [Add Note]

We'd have to test against low resolutions though.

**Comment by shubhamkmr04** [comment]
Simulator Screenshot - iPhone 15 Pro - 2024-08-27 at 12 10 18

**Comment by kaloudis** [comment]
> Simulator Screenshot - iPhone 15 Pro - 2024-08-27 at 12 10 18

this is definitely better than before, but the margins are driving me nuts. We need to find a way to make sure everything lines up. We could also use less margins on top and bottom too for this row.

**Comment by kaloudis** [comment]
@shubhamkmr04 can you provide screenshots if the layout has been changed at all?

**Comment by shubhamkmr04** [comment]
> @shubhamkmr04 can you provide screenshots if the layout has been changed at all?

Simulator Screenshot - iPhone 15 Pro - 2024-08-30 at 13 24 13

**Comment by shubhamkmr04** [comment]
Tested on iPhone 15 pro for now

File: views/SendingLightning.tsx [hunk 1]
```diff
@@ -116,7 +157,12 @@ export default class SendingLightning extends React.Component<
             payment_error,
             isIncomplete
         } = TransactionsStore;
-        const { storedNotes } = this.state;
+        const { storedNotes, currentPayment } = this.state;
+
+        const enhancedPath = currentPayment?.enhancedPath;
+
+        const isPaymentPathExist =
```
Comment (kaloudis) [inline views/SendingLightning.tsx:164]: nit: var name needs work, grammatically  you probably wanted `doesPaymentPathExist`, but `paymentPathExists` may be better. Up to you.

Comment (shubhamkmr04) [reply views/SendingLightning.tsx]: done, also fixed the bottom. Now looks good for android, small iPhones and iPad as well

**Pull request merged**